src/mono/wasm/debugger/BrowserDebugHost/Startup.cs

```diff
@@ -112,6 +85,23 @@ string GetEndpoint(HttpContext context)
                     return $"{devToolsHost.Scheme}://{devToolsHost.Authority}{request.Path}{request.QueryString}";
                 }
 
+                async Task ReturnPageLinksToDebug(HttpContext context)
+                {
+                    HttpRequest request = context.Request;
+                    Dictionary<string, string>[] tabs = await ProxyGetJsonAsync<Dictionary<string, string>[]>(GetEndpoint(context) + "json/list");
```

Comment:
This [comment](https://bugs.chromium.org/p/chromium/issues/detail?id=1232509#c12) suggests that `json/list` etc are also being deprecated. Should we get the list of pages using `chrome://inspect` somehow?

Reply:
`chrome://inspect` is still using json/list to get information. When they really change it, we will probably need to work again on it.